File: 5.py
# check if prime or not
num  = int(input("Enter #:"))
factors = 0
if num ==0 or num ==1:
    print("Neither prime nor composite")
else:
    for i in range(1,num+1):
        if num % i == 0 :
            factors += 1
    
    print("Prime" if factors == 2 else "Not Prime")

File: 5.c
// check if prime or not
#include <stdio.h>
int main(){
    int a, factors = 1;
    printf("Enter #:");
    scanf("%d",&a);
    for(int i=1;i<a;i++){
        if (a % i == 0) {
            factors++;
        }
    }
    if (factors > 2){
        printf("Is not prime");
    }
    else{
        printf("Is prime");
    }
    return 0;
}